litmus-portal/frontend/src/components/ToolTip/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { Theme, withStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+
+const ToolTip = withStyles((theme: Theme) => ({
+  tooltip: {
+    backgroundColor: theme.palette.background.paper,
+    color: theme.palette.text.primary,
+    boxShadow:
+      '0rem 0.128rem 0.384rem rgba(0,0,0,0.1), 0rem 0.682rem 1.54rem rgba(0,0,0,0.13)',
```

Comment:
can we take colours from litmus-ui theme file?

Reply:
Tooltip specific colors `rgba` are not defined in Litmus-ui

Reply:
let's try to take it from theme.shadows[<elevation>]
elevation would be 0...24 any number you seem appropriate